cmd/jaeger/config-badger.yaml

```diff
@@ -1,12 +1,20 @@
 service:
-  extensions: [jaeger_storage, jaeger_query]
+  extensions: [jaeger_storage, jaeger_query, remotesampling]
   pipelines:
     traces:
       receivers: [otlp]
       processors: [batch]
       exporters: [jaeger_storage_exporter]
 
 extensions:
+  remotesampling:
+    static:
+      file: ./cmd/jaeger/sampling-strategies.json
+      reload_interval: 1s
+    adaptive:
```

Comment:
File & adaptive sections do not make sense simultaneously, it's either one or another. Or am I missing something?

Reply:
Yes, I already added a Conditional Check, which throws an error if both are provided.

Since these are production configs, I've included both options to make sure that the user is aware of their availability.

Reply:
I have added a comment in the config, to make it more clear.

Reply:
keeping them both means the config file is invalid. Config files are supposed to work without changes.